README.md

```diff
@@ -1,6 +1,6 @@
 # `rdme` - ReadMe's CLI
```

Comment:
@erunion can you update the versions docs to reflect `--raw`? Looks great otherwise!! Happy to approve after that :)